mlir/lib/IR/MLIRContext.cpp

```diff
@@ -596,6 +596,7 @@ void MLIRContext::disableMultithreading(bool disable) {
   // Update the threading mode for each of the uniquers.
   impl->affineUniquer.disableMultithreading(disable);
   impl->attributeUniquer.disableMultithreading(disable);
+  impl->distinctAttributeAllocator.disableThreadLocalStorage(disable);
```

Comment:
Shouldn't this be disable multi-threading for consistency? I believe the result is probably the same but it seems more intuitive to use the other function here.

Reply:
Yes good spot, this is a mistake, thank you.